### PR TITLE
OSSM-3352 Add maistra-builder image for 3.0

### PR DESCRIPTION
**Pull request by dgn**
It's a verbatim copy of the 2.4 image for now. I'm also adding the respective jobs to verify the build and push it. I applied those changes to prow already so that I don't need two PRs...

### CODE REVIEW

File: prow/config/presubmits.yaml [hunk 1]
```diff
@@ -278,6 +278,39 @@ presubmits:
       - emptyDir: {}
         name: varlibdocker
 
+  - name: test-infra_build-containers-3.0
+    decorate: true
+    path_alias: github.com/maistra/test-infra
+    skip_report: false
+    run_if_changed: '^docker/maistra-builder_2\.4\.Dockerfile|^docker/scripts'
```
Comment (jwendell) [inline prow/config/presubmits.yaml]: 3.0?

**Comment by jwendell** [comment]
This is weird. Why the job test-infra_build-containers-3.0 was triggered if the config is not applied yet?

@dgn did you change prow config by hand? Plus it looks like you manually pushed the 3.0 image to the wrong place.

**Comment by dgn** [comment]
> This is weird. Why the job test-infra_build-containers-3.0 was triggered if the config is not applied yet?

I did apply it manually to not have to create two PRs. It's cheating, I know ;)

> @dgn did you change prow config by hand? Plus it looks like you manually pushed the 3.0 image to the wrong place.

Ugh, looks like the push failed on its last legs. Pushing again

**Comment by jwendell** [comment]
Remember to delete the branch when this is merged ;)

**Pull request merged**